src/modules/global_vars/__init__.py

```diff
@@ -74,5 +51,28 @@
     "workdir": {
         "mode": "priority",
         "priority": ("operator", "project", "application")
+    },
+    "naming_log_file": {
+        "mode": "priority",
+        "priority": ("operator", "project", "application")
+    },
+    "naming_codename": {
+        "mode": "priority",
+        "priority": ("operator", "project", "application")
+    },
+    "notify": {
+        "mode": "inclusive",
+        "inclusive": ("operator", "project", "application"),
+        "default": {}
+    },
+    "exclude_devices": {
```

Comment:
Field names should be adjusted to final template here: b5190492921025f4806e42db88fc8f0b8324aa6e.